Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System;
+using System.Globalization;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    public class BalanceStats
+    {
+          private const string InValidOutput = "N/A.";
+          private readonly StringBuilder _persons;
+          private readonly decimal _amount;
+          public const string culture = "en-GB";
+
+        public BalanceStats(StringBuilder persons, decimal amount)
+        {
+            _persons = persons;
+            _amount = amount;
+        }
+
+        public StringBuilder GetPersons()
+        {
+            return _persons;
+        }
+
+        public decimal GetAmount()
+        {
+            return _amount;
+        }
+
+        
+        public static Person[] FindHighestBalanceEver(string[] peopleAndBalances)
+        {
+            var parsedFile = BalanceParser.Parser(peopleAndBalances); 
+            var person = FindHighestBalance(parsedFile);
+
+            return person ;
+        }
+
+        public static Person[] FindHighestBalance(Person[] balances)
+        {
+            CultureInfo.CurrentCulture = new CultureInfo(culture);
+            var persons = new Person[0];
+            var highestBalanceEver = decimal.MinValue;
+           
+           
+            for (int i = 0; i <= balances.Length - 1; i++)
+            {
+                var currentPerson = balances[i];
+
+                var highestAmountOfPerson = currentPerson.HighestBalance(); 
+
+                if (highestAmountOfPerson > highestBalanceEver)
+                {
+                    highestBalanceEver = highestAmountOfPerson;
+                    persons = new Person[0]; 
+                   
+                }
+
+                if (highestAmountOfPerson >= highestBalanceEver)
+                {
+                    persons = AddPerson(currentPerson, persons);
+                }
+            }
+
+            return persons ;
+        }
+
+        private static Person[] AddPerson(Person person, Person[] persons)
+        {
+            var newArray = new Person[persons.Length + 1];
+            for (int i = 0; i < persons.Length ; i++)
+            {
+                newArray[i] = persons[i]; 
+            }
+
+            newArray[^1] = person;
+            return newArray; 
+        }
+
+        //    public static string FindPersonWithBiggestLoss(string[] peopleAndBalances)
+        //    {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5053435)